dwave/system/samplers/dwave_sampler.py

```diff
@@ -180,6 +182,19 @@ def __init__(self, failover=False, retry_interval=-1, **config):
         self.client = Client.from_config(**config)
         self.solver = self._get_solver(penalty=self._solver_penalty)
 
+    def close(self):
+        """Close the underlying cloud-client in order to release system
+        resources like threads.
```

Comment:
```suggestion
        """Close the underlying cloud client to release system resources such as threads.
```

Alternatively ``dwave-cloud-client``. Also elsewhere below